bugbot/gcp.py

```diff
@@ -0,0 +1,36 @@
+from typing import Iterable, Optional
+
+from google.cloud import bigquery
+from google.oauth2 import service_account
+
+from bugbot import utils
+
+SCOPES = {
+    "cloud-platform": "https://www.googleapis.com/auth/cloud-platform",
+    "drive": "https://www.googleapis.com/auth/drive",
+}
+
+
+def get_bq_client(
+    project: str, scopes: Optional[Iterable[str]] = None
+) -> bigquery.Client:
+    """Get a bigquery.Client for a given project
+
+    :param project: Name of the project.
+    :param scopes: Optional iterable containing the scopes the client should have.
+                   By default this will be the cloud-platform scopes required to
+                   run queries.
+    :returns: bigquery.Client
```

Comment:
Could you please use Google-style docstrings to be consistent with the rest of the codebase?

Example: https://github.com/mozilla/bugbot/blob/4afbef850b2bb8d22c8e926beb421fd44aac7328/bugbot/utils.py#L751-L758